lra/coordinator/client/narayana-client/src/main/java/io/helidon/lra/coordinator/client/narayana/NarayanaClient.java

```diff
@@ -260,6 +267,7 @@ public void leave(URI lraId, PropagatedHeaders headers, Participant p) {
 
     @Override
     public LRAStatus status(URI lraId, PropagatedHeaders headers) {
+        logF("Checking status of LRA " + lraId);
```

Comment:
This is wrong usage of logging - you have to guard a statement if it concatenates strings.
So either use parameters, or log directly using a guarded statement.